### ISSUE TITLE
ImportError: cannot import name Templar

### ISSUE THREAD

**Issue by rabin-io** [fabriziopandini/vagrant-playbook#9]
This is cross post from [fabriziopandini/vagrant-compose/issues/14](https://github.com/fabriziopandini/vagrant-compose/issues/14) and i open the issue here as I think the problem is related to this project and not to vagrant-compose.

Basically with sample setup (see [declarative) approche](https://github.com/fabriziopandini/vagrant-compose/blob/master/doc/declarative.md) I got this error message when trying to invoke `vagrant status` 

```
Error running cluster playbook:
Traceback (most recent call last):
  File "/usr/bin/vagrant-playbook", line 11, in <module>
    load_entry_point('vagrant-playbook==0.7.2', 'console_scripts', 'vagrant-playbook')()
  File "/usr/lib/python2.7/site-packages/vagrantplaybook/__main__.py", line 22, in main
    from vagrantplaybook.playbook.executor import Executor
  File "/usr/lib/python2.7/site-packages/vagrantplaybook/playbook/executor.py", line 9, in <module>
    from vagrantplaybook.ansible import ansible_unwrap, ansible_loader
  File "/usr/lib/python2.7/site-packages/vagrantplaybook/ansible.py", line 6, in <module>
    from ansible.template import Templar
  File "/usr/lib/python2.7/site-packages/ansible/template/__init__.py", line 47, in <module>
    from ansible.vars.unsafe_proxy import UnsafeProxy, wrap_var
  File "/usr/lib/python2.7/site-packages/ansible/vars/__init__.py", line 42, in <module>
    from ansible.template import Templar
ImportError: cannot import name Templar
```

After looking in the code of `/usr/lib/python2.7/site-packages/vagrantplaybook/ansible.py` and consulting [StackOverflow](https://stackoverflow.com/questions/9252543/importerror-cannot-import-name-x) I check for `circular dependent imports` so I changed the above file like this,

```diff
--- a/usr/lib/python2.7/site-packages/vagrantplaybook/ansible.py    2016-11-15 23:15:28.000000000 +0200
+++ b/usr/lib/python2.7/site-packages/vagrantplaybook/ansible.py 2017-07-30 22:26:13.608408470 +0300
@@ -3,9 +3,8 @@

 #TODO: remove dependencies from ansible (use default json library + ninja2)
 from ansible.utils.unicode import to_str
-from ansible.template import Templar
 from ansible.parsing.dataloader import DataLoader
-
+from ansible.template import Templar
 from vagrantplaybook.compat import compat_text_type

 ansible_tempar = Templar
```
Basically I moved the import line down by 2 lines, and that seems to resolve my problem.

**Comment by fabriziopandini** [comment]
@rabin-io thanks for your help, really appreciated!

I'm coming back today from some days OOF. If this fix works on your environment, I will make some regression tests in other platforms and then integrate your changes.

**Comment by rabin-io** [comment]
Thank you for the great application/pluging;

**Comment by fabriziopandini** [comment]
@rabin-io thanks for your contribution, that now is in release v0.7.3 published just now.
To update your environment you can use `pip install vagrant-playbook --upgrade`.
Let me know if there are still problems

**Comment by rabin-io** [comment]
Thanks, 

one small point, there is this  `DEPRECATION WARNING`  when running `vagrant-playbook`, isn't that something that should we address as we currently using Ansible 2.3 ?
```
$ vagrant-playbook -f cluster-setup.yaml
[DEPRECATION WARNING]: ansible.utils.unicode.to_str is deprecated.  Use ansible.module_utils._text.to_native instead.
This feature will be removed in version 2.4. Deprecation warnings can be disabled by setting deprecation_warnings=False 
in ansible.cfg.
```

**Comment by fabriziopandini** [comment]
Created a new issue to track the deprecation warning
